### ISSUE TITLE
Not able to add server into the app after OS upgrade

### ISSUE THREAD

**Issue by dejanvujanovic** [RocketChat/Rocket.Chat.Electron#1790]
#735  My Setup

- Operating System: Ubuntu 20.04.1 LTS (Focal Fossa)
- App Version: 3.0.1
- Installation type: snap

- [x] I have tested with the latest version
- [x] I can simulate the issue easily

## Description
After upgrading from Ubuntu 18.04 to 20.04, I am no longer able to add server into the client. As soon as I try that, I get error **_No valid server found at the URL_**. Though official rocket chat works fine. Snap currently have stable version **3.0.1** but I have also tested it on **beta** (**3.0.0**) and on **edge** (**3.0.2**), same issue. This is what I get when I type **snap changes**
```
ID   Status  Spawn                    Ready                    Summary
209  Done    yesterday at 10:23 CEST  yesterday at 10:24 CEST  Auto-refresh snap "core"
213  Done    today at 04:09 CEST      today at 04:09 CEST      Refresh all snaps: no updates
214  Done    today at 04:54 CEST      today at 04:54 CEST      Refresh "rocketchat-desktop" snap from "edge" channel
215  Done    today at 09:36 CEST      today at 09:37 CEST      Refresh "rocketchat-desktop" snap from "beta" channel
```

### Current Behavior
**_No valid server found at the URL_**

### Expected Behavior
Server added to the rocket chat desktop app.

**Comment by Sadrak** [comment]
related: #1787

**Comment by HermanCattaneo** [comment]
@dejanvujanovic @Sadrak 

Desktop client 3.0.3 solved the issue, please check: #1787 

@tassoevan I think you can close this one too.
